MozziGuts_impl_MBED.hpp

```diff
@@ -14,9 +14,47 @@
 #  error "Wrong implementation included for this platform"
 #endif
 
+#define CHUNKSIZE 64
```

Comment:
Defined common for input and output, I guess that even if we do not use it it does not take any memory as it is a macro.

Also, out of reach from standard users.